### PR TITLE
Update microsoft-defender-offline.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed the section which talks about reviewing Offline scan results as it is not longer available.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9194

File: microsoft-365/security/defender-endpoint/microsoft-defender-offline.md [hunk 1]
```diff
@@ -132,9 +132,6 @@ See the following for more information:
     > [!NOTE]
     > In Windows 10, version 1607, the offline scan could be run from under **Windows Settings** \> **Update & security** \> **Windows Defender** or from the Windows Defender client.
 
-## Review scan results
-
-Microsoft Defender Offline scan results will be listed in the [Scan history section of the Windows Security app](microsoft-defender-security-center-antivirus.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-offline.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**